README.md

```diff
@@ -241,7 +241,9 @@ Attribute key:
 
 ##### `docker`
 
-The `docker` export type writes the result image as an Docker image specification tarball https://github.com/moby/moby/blob/master/image/spec/v1.2.md on the client. Tarballs created by this exporter are also OCI compatible.
+The `docker` export type writes the single-platform result image as a Docker image specification tarball https://github.com/moby/moby/blob/master/image/spec/v1.2.md on the client. Tarballs created by this exporter are also OCI compatible.
+
+Currently, multi-platform images cannot be exported with the `docker` export type. The most common usecase for multi-platform images is to directly push to a registry (see [`registry`](#registry)).
```

Comment:
That's an assumption I do not agree to. You certainly want to run images locally before pushing. https://github.com/docker/buildx/issues/59